### ISSUE TITLE
Change location of no-server instances

### ISSUE THREAD

**Issue by Pereck** [PokemonGoMap/PokemonGo-Map#1314]
Is it possible to change the search location of the no-server instances without turn it off and turn it on with new coordinates?

**Comment by FrostTheFox** [comment]
Nope, they weren't built with location changing during runtime in mind.
